ceph_iscsi_config/lun.py

```diff
@@ -197,7 +197,7 @@ def required_features(cls):
 
 class LUN(object):
 
-    def __init__(self, logger, pool, image, size, allocating_host):
+    def __init__(self, logger, pool, image, size, allocating_host, ring_buffer_size=None):
```

Comment:
I'd avoid adding a new constructor parameter and instead would use a property attribute to set the override after it's 
constructed if it's being changed from default / the overridden value. There are too many places where the ```LUN``` object is created to update to properly handle extracting a possible override from the disk config and re-applying via the constructor.

Reply:
I see just a handful of places where `LUN` (from `ceph_iscsi_config.lun`) is instantiated or is it the snapshot pr which would be needing this?

Reply:
Yeah, I don't want my snapshot PR to break the override (since it shouldn't have to worry about re-applying an existing setting). Plus, there is also ceph-ansible that should eventually support overrides as well.